admin/views/pupil-status/pupil-status-details/error-details.ejs

```diff
@@ -0,0 +1,37 @@
+<details class="govuk-details govuk-!-margin-bottom-0" data-module="govuk-details">
+    <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">
+          Steps to take
+        </span>
+    </summary>
+    <div class="govuk-details__text">
+        <dl class="govuk-panel-list">
+            <dt>
+                <div><strong><u>Either</u> log into the device that pupil used</strong></div>
+                <div>
+                    <p>
+                        Step 1 <br/>
```

Comment:
Step 1 and Step 2 seem like they should be marked up with an ordered list

Reply:
I'l keep this as description list element as per the last commit until a change in the requirements.

Reply:
resolved